packages/password-input/README.md

```diff
@@ -0,0 +1,117 @@
+| `className`          | `string`                                                                                                                                                    | The className passed to the root element of the component.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |                |
+| ...                  | native `input` attributes                                                                                                                                   | Any other props will be spread on the root `input` element                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |                |
+
+## States
+
+```js
+const States = {
+  Error: 'error', // red x icon, red text
+  Warning: 'warning', // red warning icon, red text
+  Valid: 'valid', // green checkmark, black text
+  None: 'none', // gray checkmark, gray text
+} as const;
+
+```
+
+## Special Case
+
+### Aria Labels
+
+Either `label` or `aria-labelledby` or `aria-label` must be provided a string, or there will be a console error. This is to ensure that screenreaders have a description for what the PAssword Input does.
+
+### Aria Describedby
```

Comment:
I wonder if because of this it's worth exporting <Error /> and <Valid /> here?

Reply:
Are you talking about the `<Error />` component from `Typography`? I don't think we have a `valid` component. But I could make the `StateNotifications` component a separate component so that it could be used in a custom container.